ansible/roles/dev-desktop/files/github_app/readme.md

```diff
@@ -0,0 +1,53 @@
+These instructions are for server-side setup and debugging.
```

Comment:
We usually have all our docs in the [Documentation section of the forge](https://forge.rust-lang.org/infra/docs/index.html), could this be moved to a PR there? I'd create separate pages for team members and for the infra team (like for AWS access).

Reply:
This documentation is purely for the infra team, there's nothing there a user of the dev desktops needs to read

Reply:
Definitely! We're trying to keep the documentation on the infrastructure on the forge though, so that it's all in a central place rather than being scattered around :slightly_smiling_face:

Reply:
Oh sorry, I'm all for that. I was just confused by what you wanted me to split when you wrote

> I'd create separate pages for team members and for the infra team